geth/jail/extensions/console/console.go

```diff
@@ -0,0 +1,138 @@
+package console
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/robertkrimen/otto"
+	"github.com/status-im/status-go/geth/jail/extensions"
+	"github.com/status-im/status-go/geth/node"
+)
+
+var (
+	// Stdout defines the default writer for the console.log
+	// delivery call.
+	Stdout io.Writer = os.Stdout
```

Comment:
Generally, it's a bad practice to use global variables even if it's os.Stdout and even if Go core already uses it at some places. It hinders writing tests and running them in parallel and eventually we may end up with a lot of internal inter-dependencies for different global variables and it also may add implicit dependencies like in that case all of which will may make our code unmaintainable. So let's stick to the other practice: store state only inside objects, not packages. There's also recently been a relevant article on the topic: http://peter.bourgon.org/blog/2017/06/09/theory-of-modern-go.html

Currently, let's leave this as-is and fix it later under some refactoring task.